### DISCUSSION TITLE
分享卡

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2639] [插件说明]
描述
---

> 导出分享卡

地址
---

[https://simpread.ksria.cn/plugins/details/GS6jyEA2zA](https://simpread.ksria.cn/plugins/details/GS6jyEA2zA)

功能
---

在阅读模式将任何内容（文字，图片）生成分享卡。

分享效果
---

![image](https://user-images.githubusercontent.com/81074/128592139-a064a04d-ff8b-4cff-9c45-0cf420881b52.png)

![image](https://user-images.githubusercontent.com/81074/128592182-c5aeac20-50f8-40f6-93cf-29dd0b244be4.png)

![image](https://user-images.githubusercontent.com/81074/128592215-81c81dd7-04b6-4d08-b9ad-a37dfe4b8c0e.png)

![image](https://user-images.githubusercontent.com/81074/128592294-d8eea966-1f5b-4053-9db2-5f42d8990f50.png)

如何使用
---

选中任何内容后，通过快捷键 `s c` 或**选项页 → 控制栏 → 动作 → 插件触发器 → 分享卡** 使用

选项
---

![image](https://user-images.githubusercontent.com/81074/128592046-460447a7-0943-4588-92a1-5c4f2f76cf4e.png)

![image](https://user-images.githubusercontent.com/81074/128592060-e9622f5b-599a-440f-805f-19bb5c7aa8e3.png)

注意事项
---

使用此方式后，无法通过阅读模式 → 右下角 → 控制栏 来改变宽度。

更新日志
---

### 1.0.2

- `Optimize` 为配合 [插件管理器](https://github.com/Kenshin/simpread/discussions/4221) 的更新方式做了数据结构的优化。

### 1.0.1

- 修复因跨域造成的无法下载的错误。

- 修复当内容较多时样式的错误。

- 修复包含图片后出现的横向滚动条的问题。

- 优化分享卡导出时的字体样式。

### 1.0.0

- 快捷键 `s c`

- 选中后生成分享卡

- 设置

    - 是否添加简悦 Logo

    - 是否添加题图

    - 是否添加标题

    - 是否清除全部文本样式

    - 添加描述

    - 添加描述的样式
        - 样式一包含了不同颜色的 blockquote

    - 导出尺寸（大 /  小）

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by pikawindy** [comment]
请问为什么我的分享卡如果是附带有图片就没办法生成，纯文字就可以

**Reply by Kenshin** [reply]
嗨，截图给我看看并告知 URL

**Comment by pikawindy** [comment]
https://www.runningcheese.com/useragent
![比特截图2021-08-24-11-38-53](https://user-images.githubusercontent.com/63467634/130552084-bab307bd-6dff-48d1-a9c5-d6cb50d4d10f.png)
又试了下，发现在其他网站是可以的，难道是这网站不支持？

**Reply by Kenshin** [reply]
嗨，我测试了，确实如此。因为导出图片触发了某些安全性的限制会造成此问题。
目前没有太好的解决方案。 😂

**Reply by pikawindy** [reply]
> 嗨，我测试了，确实如此。因为导出图片触发了某些安全性的限制会造成此问题。
> 目前没有太好的解决方案。 😂

没事，算了，谢谢

**Reply by Kenshin** [reply]
嗨，这个 Bug 已经修复了，打开 https://simpread.ksria.cn/plugins/details/GS6jyEA2zA 直接安装即可搞定此问题。

**Comment by felix2049lou** [comment]
![image](https://user-images.githubusercontent.com/87223257/135264261-9bee46cc-979a-4225-b789-98e87b8c08da.png)
文章标题和引用文字基本上样式一样，很难分别谁是标题谁是引用文字

**Reply by felix2049lou** [reply]
![image](https://user-images.githubusercontent.com/87223257/135265453-619654ad-694e-4545-b7a0-18302da64ea5.png)
还有些转换问题⋯⋯到底现在都无有使用成功

**Reply by Kenshin** [reply]
嗨，确实在某些页面转换失败的情况。
